blog.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Dog Adoption for Seattle">
+    <meta name="keywords" content="Dogs, Adoption, Seattle">
+    <meta name="author" content="Keith McDade">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt A Dog</title>
+  </head>
+
+  <body>
+    <header>
+      <div>
+        <img src="images/adoptadog-logo.svg" alt="German Shepard shilouette facing right">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+      <div>
+        <img src="images/cart-icon.svg" alt="Cart Icon" >
+        <p>$1234.56</p>
+        <a href="checkout.html"> Checkout</a>
+      </div>
+    </header>
+
+    <nav>
+      <p>
+        <a href="index.html">Home</a>  /
+        <a href="dogs.html">Dogs</a>  /
+        <a href="blog.html">Blog</a>
+      </p>
+    </nav>
+
+    <section>
+      <h1>Adoptadog Blog</h1>
+      <div>
+
+        <article>
```

Comment:
I would recommend using a div instead of article tag here. You have some warnings in your html validation as both sections and articles expect a header tag inside. Articles are meant to be self contained items that make sense on their own outside of the context of the page.